lib/util.js

```diff
@@ -138,11 +139,16 @@ function cleanUpAssertion(element) {
 }
 
 function getAssertionComponents(element) {
-  let matched = element.match(/(.*)\/\/\s(\w+-?\w+?)(:\s(.+))?;/);
+  const regEx =/^(.*)\/\/\s*(?:([a-z\-]+:|throws:?))([^;]+)?(?:.*)/i;
```

Comment:
If you want to extract the comment,  what occurs after the optional `;`, the following should do the trick

```js
/^(.*)\/\/\s*(?:([a-z\-]+:|throws:?))([^;]+)?(?:;(.*))?$/
```

Reply:
The test file can ignore these comments in code samples.